app/js/reducers/addDrugOrderReducer.js

```diff
@@ -14,11 +18,19 @@ const addDrugOrderReducer = (state = initialState, action) => {
       return {
         ...state,
         addedOrder: action.data,
+        status: {
```

Comment:
Hi @Efosaok 
For `POST_DRUG_ORDER_SUCCESS`, are you not supposed to also set the `errorMessage` property to empty?

Reply:
it's not necessary, I basically use the status object to check for error or success and you see that is what I update in both `action type` cases.

Reply:
Using the status object to check for error messages would definitely work fine, but I don't just think it would be nice to still have an error message in the state while the error status is `false` and a drug order has been successfully added.

Reply:
addressed